testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/custom/CustomThemeTest.java

```diff
@@ -61,26 +58,25 @@ public void configureTestRealm(RealmRepresentation testRealm) {
     protected LoginPage loginPage;
 
     @Page
-    protected AccountUpdateProfilePage profilePage;
+    protected AppPage appPage;
```

Comment:
Is `appPage` used anywhere in this or any child class?

Reply:
Neither that one nor `loginPage`, removing them